src/redux/categories/categoriesSlice.jsx

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      const newBook = {
+        id: new Date().getTime(),
+        title: action.payload,
+      };
+      return {
+        ...state,
+        books: [...state.books, newBook],
+      };
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      return {
+        ...state,
+        books: state.books.filter((book) => book.id !== bookId),
+      };
+    },
+  },
+});
+
+export const { addBook, removeBook } = booksSlice.actions;
```

Comment:
- [ ] `categoriesSlice` is missing! You're creating another `booksSlice` here in the _categoriesSlice.jsx_ instead of `categoriesSlice`. Let's improve upon this aspect by defining the `categoriesSlice` in the _categoriesSlice.jsx_, as the guideline below:
   * Imports `createSlice` from `@reduxjs/toolkit`
   * Creates an `initialState` with a state value `categories` that contains an empty array
   * Creates 1 reducer, `checkStatus`
   * Exports by default `categoriesSlice.reducer`
   * Exports by `checkStatus` as an action